### PR TITLE
stretch background to fill the berlin container and fix margin/padding size problem

### LINKED ISSUE #6620: [Rendering] background is not stretchable
### Target Platforms

NodeJS

### SDK Version

0

### Application Name

Designer

### Problem Description

For the new added berlin config, card is fixed size. background should cover the whole card area

### Screenshots

image

### Card JSON

```JSON
n/a
```

### Sample Code Language

_No response_

### Sample Code

_No response_

### LINKED ISSUE #6646: [Designer] berlin conf: correct the margin and padding

### LINKED ISSUE #6647: [Designer] berlin conf: extend the background to header area

### LINKED ISSUE #6648: [Designer] berlin conf: In preview, truncate the content when overflow

### LINKED ISSUE #6649: [Designer] berlin conf: Add a dot button to the header area

### LINKED ISSUE #6650: [Designer] berlin conf: Round corner for the background

### PR DESCRIPTION
**Pull request by licanhua**
Adjust the outer container 4px(1+1 padding, 1+1 border), and inner container width is 300
Fix #6646 [Designer] berlin conf: correct the margin and padding
image

Fix #6648 [Designer] berlin conf: In preview, truncate the content when overflow
image
image

Fix #6620  [Rendering] background is not stretchable
Fix #6647 [Designer] berlin conf: extend the background to header area
Fix #6649 [Designer] berlin conf: Add a dot button to the header area
Fix #6650 [Designer] berlin conf: Round corner for the background

Before :
image
image

After:
image

image

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/6640)

### CODE REVIEW

File: source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container.ts [hunk 1]
```diff
@@ -37,11 +37,20 @@ export class BerlinContainer extends HostContainer {
 		header.className = "berlin-header";
 		outerFrame.appendChild(header);
 
-		const frame = document.createElement("div");
-		frame.className = "berlin-inner-container";
-		frame.appendChild(this.cardHost);
+		const headerText = document.createElement("p");
+		headerText.innerHTML = "Berlin Header";
```
Comment (paulcam206) [inline source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container.ts:41]: `innerHTML` should avoided in *almost* all circumstances. in this case, `textContent` is the more appropriate property to manipulate. it's not consequential in this instance, but it's a best practice...

if you're curious as to why, consider what happens if the string here someday gets wired up to untrusted input -- someone can inject arbitrary HTML into our payload (think `<script>`).

```suggestion
		headerText.textContent = "Berlin Header";
```

File: source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container.ts [hunk 2]
```diff
@@ -37,11 +37,20 @@ export class BerlinContainer extends HostContainer {
...
+		header.appendChild(headerText);
 
-		outerFrame.appendChild(frame);
+		const moreButton = document.createElement("button");
+		moreButton.className = "berlin-header-add-button";
+		moreButton.innerHTML = "...";
```
Comment (paulcam206) [inline source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container.ts:46]:
```suggestion
		moreButton.textContent = "...";
```

**Review by paulcam206 — CHANGES_REQUESTED** [review]

File: source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container.ts (hunk 2 above)
Comment (paulcam206) [reply source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container.ts:46]: this one's still here :)

Comment (licanhua) [reply source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container.ts]: good catch

**Review by paulcam206 — APPROVED** [review]

**Review by jwoo-msft — APPROVED** [review]

**Pull request merged**